### ISSUE TITLE
Make MAX_CONTRACT_PRELAUNCH_TIME_M configurable 

### ISSUE THREAD

**Issue by linggao** [open-horizon/anax#1955]
Currently it defaults 10 minutes, but for some services that has very large image, the agreement got canceld before the image is loaded.
Please also remove the following consts from governance.go because they are not referenced anywhere any more.
````
// TODO: make this module more aware of long-running setup operations like image downloading and dockerfile loading
// the max time we'll let a contract remain unconfigured by the provider
const MAX_CONTRACT_UNCONFIGURED_TIME_M = 20

const MAX_MICROPAYMENT_UNPAID_RUN_DURATION_M = 60

// enforced only after the workloads are running
const MAX_AGREEMENT_ACCEPTANCE_WAIT_TIME_M = 20
````

**Comment by sirotsinskuy** [comment]
Working on it

**Comment by edeediong** [comment]
I believe this issue should be closed as #1978 and #1957 claim to fix this
